hathor/wallet/resources/thin_wallet/address_history.py

```diff
@@ -24,6 +24,27 @@ class AddressHistoryResource(resource.Resource):
     def __init__(self, manager):
         self.manager = manager
 
+    def render_POST(self, request: Request) -> bytes:
```

Comment:
We also need to add the OpenAPI spec for this method. I don't mind if we only add a `TODO` comment though.

Reply:
Added the TODO